react/features/base/config/configWhitelist.js

```diff
@@ -130,6 +130,7 @@ export default [
     'doNotStoreRoom',
     'doNotFlipLocalVideo',
     'dropbox',
+    'dynamicBrandingUrl',
```

Comment:
You cannot override urls at runtime using the URL(config) parameters as this is a security issue. Sorry.